README.md

```diff
@@ -15,46 +24,56 @@ Blazing fast, [open source](https://github.com/zalando/beard), logic-less templa
 </html>
 ```
 
-What makes Beard special:
+## Requirements
+- Scala 2.10 or 2.11
+- a package manager like sbt or Maven
 
-  - simple syntax: inspired by mustache, we only use the `{` and `}` markers for tags
-  - streaming: as soon as we have something to be rendered, we are able to stream it to the browser. This gives high user perceived performance.
-  - fast: have benchmarked it against other jvm template engines, and we strive to keep it on top. Check out our [repository](https://github.com/zalando/beard) to run the benchmarks.
-  - beautiful: only using brackets for delimiters keeps it beautiful
+## Installing
+If you're using SBT, add this line to your build.sbt file:
 
-Uses ANTRL to compile the templates which makes the compilation really fast.
+    libraryDependencies += "de.zalando" %% "beard" % "0.0.6"
+
+Binaries are available from [bintray](https://bintray.com/zalando-spearheads/java/beard/0.0.2/view). Or simply
```

Comment:
These two lines should be replaced with:

    Binaries are available from [bintray](https://bintray.com/zalando/maven/beard/). Or simply add `resolvers += Resolver.bintrayRepo("zalando", "maven")` to your `build.sbt`.